package.json

```diff
@@ -44,7 +44,8 @@
   "homepage": "https://angular-maps.com",
   "peerDependencies": {
     "@angular/common": "^4.0.0 || ^2.0.0",
-    "@angular/core": "^4.0.0 || ^2.0.0"
+    "@angular/core": "^4.0.0 || ^2.0.0",
+    "js-marker-clusterer": "^1.0.0"
```

Comment:
That should be possible, what name would you suggest? I will try with @agm/cluster, if this is okay with you